opa-manager-operator/config.yaml

```diff
@@ -0,0 +1,107 @@
+# Default values for opa-operator.
+options:
+  replicaCount:
+    description: |
+      Desired number of pods, leaderElection will be enabled if this is greater than 1
+    type: int
+    default: 1
+
+  imagePath:
+    type: string
+    description: |
+      The location of the default image to use,
+      e.g. "openpolicyagent/gatekeeper:v3.2.3".
+    default: "openpolicyagent/gatekeeper:v3.2.3"
+
+  imagePullPolicy:
+    type: string
+    description: |
+      Image pull policy.
```

Comment:
The description should include valid values for the config.

Reply:
This description still needs to be updated.

Reply:
not sure where these changes got lost, updated